pyth-sdk/src/price.rs

```diff
@@ -207,15 +215,17 @@ impl Price {
             price,
             conf,
             expo: self.expo,
+            publish_time: self.publish_time.min(other.publish_time),
```

Comment:
I wonder if `timestamp` is a simpler name?

Reply:
`timestamp` is a type and does not have a meaning associated with it (it implicitly it means `time`). I like keeping `publish_time`.